templates/launch2.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Let's review this new design before merge into main branch. 

Did we test with multiple screens?